src/Commands/Dashboard/ViewCommand.php

```diff
@@ -26,22 +27,34 @@ class ViewCommand extends TerminusCommand implements SiteAwareInterface
      * @option string $site_env Site & environment in the format `site-name.env`
      * @option boolean $print Print URL only
      *
+     * @param null $site_env
+     * @param false[] $options
+     *
      * @return string|null
      *
      * @usage Opens browser to user's account on the Pantheon Dashboard.
      * @usage --print Prints the URL for user's account on the Pantheon Dashboard.
      * @usage <site> Opens browser to <site> on the Pantheon Dashboard.
      * @usage <site>.<env> Opens browser to <site>'s <env> environment on the Pantheon Dashboard.
+     *
+     * @throws \Pantheon\Terminus\Exceptions\TerminusException
+     * @throws \Pantheon\Terminus\Exceptions\TerminusNotFoundException
      */
     public function view($site_env = null, $options = ['print' => false,])
     {
-        list($site, $env) = $this->getOptionalSiteEnv($site_env);
-        $url = isset($site)
-            ? isset($env) ? $env->dashboardUrl() : $site->dashboardUrl()
-            : $this->session()->getUser()->dashboardUrl();
+        if (!$site_env) {
+            $url = $this->session()->getUser()->dashboardUrl();
+        } elseif ($env = $this->getOptionalEnv($site_env)) {
```

Comment:
It's better for readability to avoid assignments inside if / else. Maybe refactor to a separate small routine that can exit early with `return $this->session()->getUser()->dashboardUrl();` and avoid the `else` altogether.

Reply:
fixed